docs/workers.md

```diff
@@ -384,38 +384,38 @@ stream_writers:
 
 ##### The `typing` stream
 
-The following endpoints should be routed directly to the workers configured as
-stream writers for the `typing` stream:
+The following endpoints should be routed directly to a single worker configured
+as stream writers for the `typing` stream:
```

Comment:
Would it make more sense to modify the paragraph under `Stream writers` to say that all stream writers can only go to a single instance, except where documented?

Reply:
yes, that would probably be the better option. i change it.